html-web-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-web-page</title>
+</head>
+
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality and most unique jellies on a monthly basis.
+    For the jelly lover in your life (or yourself!), this is the gift that keeps on giving all year round. We offer three tiers of monthly jelly joy, and each one offers their own perks. Here is a breakdown of each as well as your monthly price:
+  </p>
+  <table border="2">
+  <thead>
+    <tr>
+      <th></th>
+      <th>Basic Plan</th>
+      <th>Deluxe</th>
+      <th>Jelly Giant Plan</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Three new curated jellies per month</td>
+      <th>&check;</th>
```

Comment:
each of these elements should be `td` s not `th`s those are only for the headings in the head generally speaking.